packages/amplify-category-auth/resources/adminAuth/admin-queries-api-template.json.ejs

```diff
@@ -290,6 +290,10 @@
         "ApiName": {
           "Description": "API Friendly name",
           "Value": "AdminQueries"
+        },
+        "ApiId": {
+          "Description": "API Id",
+          "Value": "AdminQueriesApiId"
```

Comment:
Should this be the `AdminQueries`'s API ID?

Reply:
Can you please update this to `{"Ref": "AdminQueries"}`. I chatted with @kaustavghosh06, and I think that's the value we want here.

Reply:
Sure. You mean replace line 296 with `{"Ref": "AdminQueries"}` right?

Reply:
Yes, that entire line would be:

```
"Value": {"Ref": "AdminQueries"}
```